src/main/java/com/fastcampus/projectboard/domain/ArticleComment.java

```diff
@@ -1,13 +1,58 @@
 package com.fastcampus.projectboard.domain;
 
+import lombok.Getter;
+import lombok.ToString;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
 import java.time.LocalDateTime;
+import java.util.Objects;
 
+@Getter
+@ToString
+@Table(indexes = {
+        @Index(columnList = "content"),
+        @Index(columnList = "createdAt"),
+        @Index(columnList = "createdBy")
+})
+@EntityListeners(AuditingEntityListener.class)
+@Entity
 public class ArticleComment {
-    private Article article;
-    private String content;
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(optional = false) private Article article;
+    @Column(nullable = false,length = 500) private String content;
```

Comment:
저는 여기서 줄바꿈을 맥락 단위로 적용하면 좀 더 보기 명료하지 않을까 싶습니다.

```suggestion
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne(optional = false) private Article article;
    @Column(nullable = false,length = 500) private String content;
```

Reply:
감사합니다!